src/Forms/Shared/Samples/Data/EditFeatureAttachments/EditFeatureAttachments.xaml.cs

```diff
@@ -51,25 +51,32 @@ public EditFeatureAttachments()
             Initialize();
         }
 
-        private void Initialize()
+        private async void Initialize()
         {
-            // Create the map with streets basemap.
-            MyMapView.Map = new Map(BasemapStyle.ArcGISStreets);
+            try
+            {
+                // Create the map with streets basemap.
+                MyMapView.Map = new Map(BasemapStyle.ArcGISStreets);
 
-            // Create the feature table, referring to the Damage Assessment feature service.
-            ServiceFeatureTable damageTable = new ServiceFeatureTable(new Uri(FeatureServiceUrl));
```

Comment:
Why isn't this sample using the `ServiceGeodatabase`?

Reply:
@nCastle1 There is an issue with the data that is being used with this sample that is causing an error when using `ServiceGeodatabase`. The issue has been logged for [it](https://devtopia.esri.com/runtime/common-samples/issues/3379).